share/man/man9/cdefs.9

```diff
@@ -0,0 +1,297 @@
+.It Sy __fastcall Ta
+.It Sy __result_use_check Ta
+.It Sy __result_use_or_ignore_check Ta
+.It Sy __returns_twice Ta Returns multiple times, like
+.Xr fork 2 .
+.It Sy __unreachable Ta This code is not reachable at runtime.
+.It Sy __predict_true(x) Ta Hint to the compiler that
+.Fa x
+is true most of the time.
+Should only be used when performance is improved for a frequently called bit of code.
+.It Sy __predict_false(x) Ta Hint to the compiler that
+.Fa x
+is false most of the time.
+Should only be used when performance is improved for a frequently called bit of code.
+.It Sy __null_sentinel Ta
+.It Sy __exported Ta
+.It Sy __hidden Ta
+.It Sy __printflike(fmtarg,firstvararg) Ta
+.It Sy __scanflike(fmtarg,firstvararg) Ta
+.It Sy __format_arg(f) Ta
+.It Sy __strfmonlike(fmtarg,firstvararg) Ta
+.It Sy __strtimelike(fmtarg,firstvararg) Ta
+.It Sy __printf0like(fmtarg,firstvararg) Ta
+.It Sy __strong_reference(sym,aliassym) Ta
+.It Sy __weak_reference(sym,alias) Ta
+.It Sy __warn_references(sym,msg) Ta
+.It Sy __sym_compat(sym,impl,verid) Ta
+.It Sy __sym_default(sym,impl,verid) Ta
+.It Sy __GLOBAL(sym) Ta
+.It Sy __WEAK(sym) Ta
+.It Sy __DECONST(type, var) Ta
+.It Sy __DEVOLATILE(type, var) Ta
+.It Sy __DEQUALIFY(type, var) Ta
+.It Sy __RENAME(x) Ta
+.El 
+.Ss Emulated Keywords
+As C evolves, many of the old macros we once used have been incorporated into
+the standard language.
+As this happens, we add support for these keywords as macros for older
+compilation environments.
+Sometimes this results in a nop in the older environment.
+.Bl -column ".Sy Keyword" ".Sy Description"
+.It Sy Keyword Ta Sy Description
+.It Sy _Alignas(x) Ta
+.It Sy _Alignof(x) Ta
+.It Sy _Noreturn Ta Expandsd to
+.Dq [[noreturn]]
+in C++-11 and newer compilation environments, otherwise
+.Dq __dead2
+.It _Static_assert(x, y) Ta Compile time assertion that
+.Fa x
+is true, otherwise emit
+.Fa y
+as the error message.
+.It Sy _Thread_local Ta Designate variable as thread local storage.
+.It Sy __generic Ta implement _Generic-like features which aren't entirely possible to emulate the _Generic keyword.
+.It Sy __noexcept Ta to emulate the C++11 argument-less noexcept form.
+.It Sy __noexcept_if Ta to emulate the C++11 conditional noexcept form.
+.El
+.Ss Support Macros
+The following macros are defined, or have specific values, to denote certain
+things about the build environment.
+.Bl -column ".Sy Macro" ".Sy Description"
+.It Sy Macro Ta Sy Description
+.It Sy __LONG_LONG_SUPPORTED Ta Variables may be declared
+.Dq long long .
+This is defined for C99 or newer and C++ environments.
+.It Sy __STDC_LIMIT_MACROS Ta
+.It Sy __STDC_CONSTANT_MACROS Ta
+.El
+.Ss Nice to have macros
+.Bl -column ".Sy Macro" ".Sy Description"
+.It Sy Macro Ta Sy Description
+.It Sy __offsetof(type,field) Ta
+.It Sy __rangeof(type,start,end) Ta
+.It Sy __containerof(x,s,m) Ta
+.El
+.Ss ID Strings
+This section is deprecated, but is kept around because too much contrib software
+still uses these.
```

Comment:
```suggestion
Contributed (contribt) software should cease using these deprecated strings.
```

Reply:
This suggestion completely changes the meaning.